src/OAuth2/AbstractProvider.php

```diff
@@ -27,6 +27,8 @@ abstract class AbstractProvider extends AbstractBaseProvider
 
     protected bool $pkce = false;
 
+    protected int $pkceCodeVerifierLength = 96;
```

Comment:
Can we rename this to `pkceCodeVerifierByteLength` to me more clear about what length it indicates?

Reply:
Yes, that will make it more understandable. I also renamed it in the `generatePKCECodeVerifier` method.